FFXIVClientStructs/FFXIV/Client/UI/AddonActionBarBase.cs

```diff
@@ -5,28 +5,33 @@ namespace FFXIVClientStructs.FFXIV.Client.UI;
 [StructLayout(LayoutKind.Explicit, Size = 0x248)]
 public unsafe partial struct AddonActionBarBase {
     [FieldOffset(0x00)] public AtkUnitBase AtkUnitBase;
-
+    
     [FieldOffset(0x220)] public ActionBarSlot* ActionBarSlots;
```

Comment:
This is most likely a StdVector<ActionBarSlot>

Reply:
Yeah, that tracks, I can throw that change in. What new name to give the replacement StdVector, though? The name `ActionBarSlots` is still accurate and clear, it's just the type that would need changing. Hmm...

Reply:
`ActionBarSlotVector`